addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -629,6 +661,90 @@ QUnit.test('composer text input cleared on message post', async function (assert
     );
 });
 
+QUnit.test('composer inputs cleared on message post in extended composer', async function (assert) {
```

Comment:
Same remark as before: don't use "extended" and instead mean mailing channel composer